src/hpttree.c

```diff
@@ -418,8 +418,10 @@ void buildAreaTree(s_area * area)
 void usage(void)
 {
     printf(
-        "Usage: hpttree [options] [areaname ...]\n" "Options:  -p - toggle pseudographics mode\n"
-                                                    "\t  -d <num>\t- for last <num> days\n");
+        "Usage: hpttree [-c config] [options] [areaname ...]\n"
```

Comment:
Please add "-c" to the options and change appropriately the man pages for hptlink and hpttree.

Reply:
Sure. I'll update man pages. Thanks for comment!